nototools/noto_cmap_phase3.xml

```diff
@@ -106,7 +106,7 @@
     <tr>Prti,Inscriptional Parthian,34,0000 000d 0020 00a0 10b40-10b55 10b58-10b5f,-1,</tr>
     <tr>Rjng,Rejang,58,0000 000d 0020 002c 002e 003a 00a0 00d7 200b-200d 2012-2015 2022 25cc 25fb-25fe a930-a953 a95f,-1,</tr>
     <tr>Runr,Runic,93,0000 000d 0020 00a0 16a0-16f8,-1,</tr>
-    <tr>SYM2,Symbols2,1880,0000-0020 0023 002a 0030-0039 007f-00a0 20e3 2400-2426 2440-244a 2700-2704 2706-2709 270c-2727 2729-274b 274d 274f-2752 2756-2794 2798-27af 27b1-27be 2800-28ff 2b00-2b57 2b59-2b73 2b76-2b95 2b98-2bb9 2bbd-2bc8 2bca-2bd1 2bec-2bef 4dc0-4dff fff9-fffb 10140-1018e 10190-1019b 101a0 101d0-101fd 102e0-102fb 10e60-10e7e 1d300-1d356 1d360-1d371 1f000-1f02b 1f030-1f093 1f0a0-1f0ae 1f0b1-1f0bf 1f0c1-1f0cf 1f0d1-1f0f5 1f321-1f32c 1f336 1f37d 1f394-1f39f 1f3cb-1f3ce 1f3d4-1f3df 1f3f1-1f3f3 1f3f5-1f3f7 1f43f 1f441 1f4fd-1f4fe 1f53e-1f54a 1f54f 1f568-1f579 1f57b-1f594 1f597-1f5a3 1f5a5-1f5fa 1f650-1f67f 1f6c6-1f6cb 1f6cd-1f6cf 1f6e0-1f6ea 1f6f0-1f6f3 1f780-1f7d4 1f800-1f80b 1f810-1f847 1f850-1f859 1f860-1f887 1f890-1f8ad 1f93b 1f946,-1,</tr>
+    <tr>SYM2,Symbols2,1972,0000-0020 0023 002a 0030-0039 007f-00a0 20e3 23f1-23f3 2400-2426 2440-244a 2700-2704 2706-2709 270c-2727 2729-274b 274d 274f-2753 2756-2794 2798-27af 27b1-27be 2800-28ff 2b00-2b57 2b59-2b73 2b76-2b95 2b98-2bb9 2bbd-2bc8 2bca-2bd1 2bec-2bef 4dc0-4dff fff9-fffb 10140-1018e 10190-1019b 101a0 101d0-101fd 102e0-102fb 10e60-10e7e 1d300-1d356 1d360-1d371 1f000-1f02b 1f030-1f093 1f0a0-1f0ae 1f0b1-1f0bf 1f0c1-1f0cf 1f0d1-1f0f5 1f30d-1f30f 1f315 1f31c 1f321-1f32c 1f336 1f378 1f37d 1f393-1f39f 1f3a7 1f3ac-1f3ae 1f3c2 1f3c4 1f3c6 1f3ca-1f3ce 1f3d4-1f3e0 1f3ed 1f3f1-1f3f3 1f3f5-1f3f7 1f408 1f415 1f41f 1f426 1f43f 1f441-1f442 1f446-1f449 1f44d-1f44e 1f453 1f46a 1f47d 1f4a3 1f4b0 1f4b3 1f4bb 1f4bf 1f4cb 1f4da 1f4df 1f4e4-1f4e6 1f4ea-1f4ed 1f4f7 1f4f9-1f4fb 1f4fd-1f4fe 1f508 1f50d 1f512-1f513 1f53e-1f54a 1f54f-1f579 1f57b-1f594 1f597-1f5a3 1f5a5-1f5fa 1f650-1f67f 1f687 1f68d 1f691 1f694 1f698 1f6ad 1f6b2 1f6b9-1f6ba 1f6bc 1f6c6-1f6cb 1f6cd-1f6cf 1f6e0-1f6ea 1f6f0-1f6f3 1f780-1f7d4 1f800-1f80b 1f810-1f847 1f850-1f859 1f860-1f887 1f890-1f8ad 1f93b 1f946,-1,</tr>
```

Comment:
These are generated by code and are not maintained by hand.  There are tools that do this comparison.  The output for this change is:

base: 2016-08-11 noto_cmap_reqs mergefile=mti_cmaps_160811.xml
target: 2016-09-14 noto_cmap_reqs mergefile=noto_cmap_phase3.xml
SYM2 # Symbols2
  added (92): 23f1-23f3 2753 1f30d-1f30f 1f315 1f31c 1f378 1f393 1f3a7 1f3ac-1f3ae 1f3c2 1f3c4 1f3c6 1f3ca 1f3e0 1f3ed 1f408 1f415 1f41f 1f426 1f442 1f446-1f449 1f44d-1f44e 1f453 1f46a 1f47d 1f4a3 1f4b0 1f4b3 1f4bb 1f4bf 1f4cb 1f4da 1f4df 1f4e4-1f4e6 1f4ea-1f4ed 1f4f7 1f4f9-1f4fb 1f508 1f50d 1f512-1f513 1f550-1f567 1f687 1f68d 1f691 1f694 1f698 1f6ad 1f6b2 1f6b9-1f6ba 1f6bc
    Miscellaneous Technical
      23f1 Zyyy So 6.0 STOPWATCH
      23f2 Zyyy So 6.0 TIMER CLOCK
      23f3 Zyyy So 6.0 HOURGLASS WITH FLOWING SAND
    Dingbats
      2753 Zyyy So 6.0 BLACK QUESTION MARK ORNAMENT
    Miscellaneous Symbols and Pictographs
     1f30d Zyyy So 6.0 EARTH GLOBE EUROPE-AFRICA
     1f30e Zyyy So 6.0 EARTH GLOBE AMERICAS
     1f30f Zyyy So 6.0 EARTH GLOBE ASIA-AUSTRALIA
     1f315 Zyyy So 6.0 FULL MOON SYMBOL
     1f31c Zyyy So 6.0 LAST QUARTER MOON WITH FACE
     1f378 Zyyy So 6.0 COCKTAIL GLASS
     1f393 Zyyy So 6.0 GRADUATION CAP
     1f3a7 Zyyy So 6.0 HEADPHONE
     1f3ac Zyyy So 6.0 CLAPPER BOARD
     1f3ad Zyyy So 6.0 PERFORMING ARTS
     1f3ae Zyyy So 6.0 VIDEO GAME
     1f3c2 Zyyy So 6.0 SNOWBOARDER
     1f3c4 Zyyy So 6.0 SURFER
     1f3c6 Zyyy So 6.0 TROPHY
     1f3ca Zyyy So 6.0 SWIMMER
     1f3e0 Zyyy So 6.0 HOUSE BUILDING
     1f3ed Zyyy So 6.0 FACTORY
     1f408 Zyyy So 6.0 CAT
     1f415 Zyyy So 6.0 DOG
     1f41f Zyyy So 6.0 FISH
     1f426 Zyyy So 6.0 BIRD
     1f442 Zyyy So 6.0 EAR
     1f446 Zyyy So 6.0 WHITE UP POINTING BACKHAND INDEX
     1f447 Zyyy So 6.0 WHITE DOWN POINTING BACKHAND INDEX
     1f448 Zyyy So 6.0 WHITE LEFT POINTING BACKHAND INDEX
     1f449 Zyyy So 6.0 WHITE RIGHT POINTING BACKHAND INDEX
     1f44d Zyyy So 6.0 THUMBS UP SIGN
     1f44e Zyyy So 6.0 THUMBS DOWN SIGN
     1f453 Zyyy So 6.0 EYEGLASSES
     1f46a Zyyy So 6.0 FAMILY
     1f47d Zyyy So 6.0 EXTRATERRESTRIAL ALIEN
     1f4a3 Zyyy So 6.0 BOMB
     1f4b0 Zyyy So 6.0 MONEY BAG
     1f4b3 Zyyy So 6.0 CREDIT CARD
     1f4bb Zyyy So 6.0 PERSONAL COMPUTER
     1f4bf Zyyy So 6.0 OPTICAL DISC
     1f4cb Zyyy So 6.0 CLIPBOARD
     1f4da Zyyy So 6.0 BOOKS
     1f4df Zyyy So 6.0 PAGER
     1f4e4 Zyyy So 6.0 OUTBOX TRAY
     1f4e5 Zyyy So 6.0 INBOX TRAY
     1f4e6 Zyyy So 6.0 PACKAGE
     1f4ea Zyyy So 6.0 CLOSED MAILBOX WITH LOWERED FLAG
     1f4eb Zyyy So 6.0 CLOSED MAILBOX WITH RAISED FLAG
     1f4ec Zyyy So 6.0 OPEN MAILBOX WITH RAISED FLAG
     1f4ed Zyyy So 6.0 OPEN MAILBOX WITH LOWERED FLAG
     1f4f7 Zyyy So 6.0 CAMERA
     1f4f9 Zyyy So 6.0 VIDEO CAMERA
     1f4fa Zyyy So 6.0 TELEVISION
     1f4fb Zyyy So 6.0 RADIO
     1f508 Zyyy So 6.0 SPEAKER
     1f50d Zyyy So 6.0 LEFT-POINTING MAGNIFYING GLASS
     1f512 Zyyy So 6.0 LOCK
     1f513 Zyyy So 6.0 OPEN LOCK
     1f550 Zyyy So 6.0 CLOCK FACE ONE OCLOCK
     1f551 Zyyy So 6.0 CLOCK FACE TWO OCLOCK
     1f552 Zyyy So 6.0 CLOCK FACE THREE OCLOCK
     1f553 Zyyy So 6.0 CLOCK FACE FOUR OCLOCK
     1f554 Zyyy So 6.0 CLOCK FACE FIVE OCLOCK
     1f555 Zyyy So 6.0 CLOCK FACE SIX OCLOCK
     1f556 Zyyy So 6.0 CLOCK FACE SEVEN OCLOCK
     1f557 Zyyy So 6.0 CLOCK FACE EIGHT OCLOCK
     1f558 Zyyy So 6.0 CLOCK FACE NINE OCLOCK
     1f559 Zyyy So 6.0 CLOCK FACE TEN OCLOCK
     1f55a Zyyy So 6.0 CLOCK FACE ELEVEN OCLOCK
     1f55b Zyyy So 6.0 CLOCK FACE TWELVE OCLOCK
     1f55c Zyyy So 6.0 CLOCK FACE ONE-THIRTY
     1f55d Zyyy So 6.0 CLOCK FACE TWO-THIRTY
     1f55e Zyyy So 6.0 CLOCK FACE THREE-THIRTY
     1f55f Zyyy So 6.0 CLOCK FACE FOUR-THIRTY
     1f560 Zyyy So 6.0 CLOCK FACE FIVE-THIRTY
     1f561 Zyyy So 6.0 CLOCK FACE SIX-THIRTY
     1f562 Zyyy So 6.0 CLOCK FACE SEVEN-THIRTY
     1f563 Zyyy So 6.0 CLOCK FACE EIGHT-THIRTY
     1f564 Zyyy So 6.0 CLOCK FACE NINE-THIRTY
     1f565 Zyyy So 6.0 CLOCK FACE TEN-THIRTY
     1f566 Zyyy So 6.0 CLOCK FACE ELEVEN-THIRTY
     1f567 Zyyy So 6.0 CLOCK FACE TWELVE-THIRTY
    Transport and Map Symbols
     1f687 Zyyy So 6.0 METRO
     1f68d Zyyy So 6.0 ONCOMING BUS
     1f691 Zyyy So 6.0 AMBULANCE
     1f694 Zyyy So 6.0 ONCOMING POLICE CAR
     1f698 Zyyy So 6.0 ONCOMING AUTOMOBILE
     1f6ad Zyyy So 6.0 NO SMOKING SYMBOL
     1f6b2 Zyyy So 6.0 BICYCLE
     1f6b9 Zyyy So 6.0 MENS SYMBOL
     1f6ba Zyyy So 6.0 WOMENS SYMBOL
     1f6bc Zyyy So 6.0 BABY SYMBOL
Zsym # Symbols
  added (3): 23e9-23ea 1f610
    Miscellaneous Technical
      23e9 Zyyy So 6.0 BLACK RIGHT-POINTING DOUBLE TRIANGLE
      23ea Zyyy So 6.0 BLACK LEFT-POINTING DOUBLE TRIANGLE
    Emoticons
     1f610 Zyyy So 6.0 NEUTRAL FACE
  removed (2): 23f1-23f2
    Miscellaneous Technical
      23f1 Zyyy So 6.0 STOPWATCH (in SYM2, Zsye)
      23f2 Zyyy So 6.0 TIMER CLOCK (in SYM2, Zsye)